snuba/query/__init__.py

```diff
@@ -260,6 +260,7 @@ def transform_expressions(
         self,
         func: Callable[[Expression], Expression],
         skip_transform_condition: bool = False,
+        skip_array_join: bool = False,
```

Comment:
Why do we need to skip this in the general case ?

Reply:
False means we don't skip the array join.